main_logic/tts_client.py

```
@@ -448,15 +468,18 @@ async def receive_messages():
                         
                     except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)